taipy/gui/data/data_accessor.py

```diff
@@ -132,17 +152,27 @@ def _register(self, cls: t.Type[_DataAccessor]) -> None:
                 for cl in classes:
                     self.__access_4_type[cl] = inst  # type: ignore
 
-    def __get_instance(self, value: _TaipyData) -> _DataAccessor:  # type: ignore
+
+    def __get_instance(self, value: _TaipyData) -> _DataAccessor:
         value = value.get() if isinstance(value, _TaipyData) else value
         access = self.__access_4_type.get(type(value))
         if access is None:
             if value is not None:
+                transformed_value = self.__gui.handle_invalid_data(value)
+                if transformed_value is not None:
```

Comment:
here we should check the availability of an accessor with the new type and return it.